packer/systems/sles/sles.lua

```diff
@@ -0,0 +1,20 @@
+function command(autoinst, other)
```

Comment:
There should be not "other" parameter in the sles version if you don't pass is when calling the function